docs/kyma/docs/029-details-install-subcomponents.md

```diff
@@ -43,4 +43,24 @@ data:
   azure-broker.enabled: "true"
 ```
 
->**NOTE:** Some subcomponents can require additional configuration to work properly.
\ No newline at end of file
+>**NOTE:** Some subcomponents can require additional configuration to work properly.
+
+## Specify subcomponents versions
+
+Versions of the Kyma components are specified in `values.yaml` file in charts. Each component version is described by two properties: `version` and `dir`. The first defines actual docker image tag. The second describes directory under which the tagged image is pushed follwed by a `/` or this value can be empty.
+
+Possible values of `dir` property:
+- `pr/` - contains images built from Pull Request
```

Comment:
- Pls remove `- ` in all those bullet points
- from Pull Request -> `from the pull request`